src/main/resources/application.yaml

```diff
@@ -95,3 +95,12 @@ feign:
     headers:
       - authorization
       - serviceauthorization
+
+amqp:
+  host: ${AMQP_HOST:rd-servicebus-sandbox.servicebus.windows.net}
+  username: ${AMQP_USERNAME:SendAndListenSharedAccessKey}
```

Comment:
can you pls update to CWRD_MQ_*

Reply:
Addressed here: https://github.com/hmcts/rd-caseworker-ref-api/pull/59/commits/18aa3b5fbb2576ab6db6998481df515fd25479ea